pkg/config/mcg.go

```diff
@@ -0,0 +1,58 @@
+package config
+
+import (
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type MCG struct {
+	config ProductConfig
+}
+
+func NewMCG(config ProductConfig) *MCG {
+	return &MCG{config: config}
+}
+
+func (m *MCG) GetWatchableCRDs() []runtime.Object {
+	return []runtime.Object{}
+}
```

Comment:
I'm unsure what we would watch here - open to suggestions. The NooBaa object would be the obvious candidate but it get's updated frequently by NooBaa operator with new timestamps - this would trigger RHOAM reconcile more often than needed. My only other thought would be the ObjectBucketClaim to try queue up a reconcile when bucket is created.

Reply:
it's ok, maybe we could improve later, if needed